akka-components/src/test/scala/ClusterComponentSpec.scala

```diff
@@ -255,7 +255,42 @@ class ClusterComponentSpec extends ScalaTestWithActorTestKit(ConfigFactory.parse
 
         val component = new ComponentObject.Component
         ComponentObject.init(component).delayedInit()
-        component.entityRefFor(ComponentObject.EntityId("a", "b"))
+        val entityRef = component.entityRefFor(ComponentObject.EntityId("a", "b"))
+        entityRef.entityId shouldBe """{"id1":"a","id2":"b"}"""
+
+        val entityRefWithPlaceholder = component.entityRefFor(ComponentObject.EntityId("a|b", "c|d"))
+        entityRefWithPlaceholder.entityId shouldBe """{"id1":"a❘❘bar❘❘b","id2":"c❘❘bar❘❘d"}"""
```

Comment:
this tests only the serialization? testing the deserialization would be great too

Reply:
added codec tests